backend/index.ts

```diff
@@ -46,10 +47,8 @@ app.use(setupStaticContent())
 app.use(setupWebSession())
 app.use(setupAuth({ oauthApi: apis.oauthApi, tokenVerificationApi: apis.tokenVerificationApi }))
 
-app.use('/redirect*', (req, res, next) => {
-  logError(req.originalUrl, 'Redirect request recieved - base url = ', req.baseUrl.substr(9))
-  next()
-})
+// The redirect base URL is defined in the values*.yaml
+app.use('/redirect*', deprecatedUrlPage(`/redirect`))
```

Comment:
Currently this is just defined in values-dev.yaml - i will promote to the prod env's if/when we get this working